### PR TITLE
Yard doc part deux

### PR DESCRIPTION
**Pull request by prpetten**
Finalizing initial YARD documetation

### CODE REVIEW

File: lib/name_drop/client.rb [hunk 1]
```diff
@@ -1,53 +1,92 @@
...
 # Namespace for classes and modules that handle interaction with Mention API
+#
 # @since 0.1.0
 module NameDrop
   # Class that encapsulates calls to Mention API
+  #
   # @since 0.1.0
   class Client
+    # Initializes new NameDrop::Client object
     def initialize(*)
```
Comment (adambedford) [inline lib/name_drop/client.rb:11]: Since we don't do anything with the arguments here, this can be removed.

**Comment by ridiculous** [comment]
👍

File: lib/name_drop/client.rb (hunk 1 above)
Comment (prpetten) [reply lib/name_drop/client.rb:11]: @adambedford Removed as per your recommendation.

File: lib/name_drop/client.rb [hunk 2]
```diff
@@ -1,53 +1,91 @@
...
 
 # Namespace for classes and modules that handle interaction with Mention API
+#
 # @since 0.1.0
 module NameDrop
   # Class that encapsulates calls to Mention API
+  #
   # @since 0.1.0
   class Client
     def initialize(*)
```
Comment (adambedford) [inline lib/name_drop/client.rb:11]: The method definition itself can be removed. perhaps that belongs in a separate PR though
